cmd/flux/testdata/diff-kustomization/diff-with-drifted-key-sops-secret.golden

```diff
@@ -6,7 +6,7 @@
 ► Secret/default/podinfo-token-77t89m9b67 drifted
 
 data
-  - one map entry removed:     + one map entry added:
-    drift-key: "*****"           token: "*****"
+- one map entry removed:   + one map entry added:
+drift-key: "*****"           token: "*****"
```

Comment:
This is due to https://github.com/homeport/dyff/releases/tag/v1.8.0

Doesn't look very good to me :/